plugins/apbsplugin.py

```diff
@@ -1667,10 +1667,20 @@ def show_error(message):
         # copied from WLD code
         sel = "((%s) or (neighbor (%s) and hydro))" % (
             self.selection.getvalue(), self.selection.getvalue())
-        self.fixColumns(sel)
-        pymol.cmd.save(pdb_filename, sel)
+
+        apbs_sel = pymol.cmd.get_unused_name()
+        apbs_clone = pymol.cmd.get_unused_name()
```

Comment:
calling get_unused_name() twice will return the same name, unless the name is actually used before the second call. In this case, the named selection is redundant and not needed, simply do: `pymol.cmd.create(apbs_clone, sel)`